### DISCUSSION TITLE
Decompiler package linked file shows up in VS

### DISCUSSION THREAD

**Discussion by Kuinox** [icsharpcode/ILSpy#2818] [General] [answered]
Hello,
Why does ILSpy decompiler projects have these links to this files ?  
https://github.com/icsharpcode/ILSpy/blob/42668e810afa2954a3fc3528f6b45a7e191c2a9c/ICSharpCode.Decompiler/ICSharpCode.Decompiler.csproj#L678

VS shows up these files in dependents projects:  
![image](https://user-images.githubusercontent.com/18743295/198308818-25cb25ed-6222-4263-87ea-e3a5a9f9b355.png)

You can get rid of these files by doing this:  
`<PackageReference Include="ICSharpCode.Decompiler" Version="8.0.0.7106-preview2" ExcludeAssets="contentfiles" />`  
But it may break ILSpy if it rely on contentfiles it ships and I have yet to test this.

**Comment by dgrunwald** [comment]
To show the documentation in a relevant place in the project browser. But that file should have type `<None>`, not `<Content>`.
I didn't realize content shows in dependent projects that way.

**Comment by christophwille** [comment]
That is a problem that seemingly came up with 8 when we switched away from .nuspec to "implicit" package creation:

https://nuget.info/packages/ICSharpCode.Decompiler/7.2.1.6856
![image](https://user-images.githubusercontent.com/344208/198341984-cd5e9054-3002-422f-8880-e769e5c52805.png)

vs

https://nuget.info/packages/ICSharpCode.Decompiler/8.0.0.7106-preview2
![image](https://user-images.githubusercontent.com/344208/198341947-7184e069-d69e-47a9-add6-e626a9358dba.png)

**Reply by Kuinox** [reply]
Do you want I open an issue for that or ?

**Comment by christophwille** [comment, marked as answer]
Fixed in https://github.com/icsharpcode/ILSpy/commit/01b27084e81936f829babc7673f2adb054ff2219

**Reply by Kuinox** [reply]
Thanks !
